utils/docker/vcluster/daos-cm.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
 
-set -o pipefail
+# shellcheck disable=SC2034,SC2145,SC2086,SC2068
+
+set -e -o pipefail
 
 VERSION=0.2
+# shellcheck disable=SC2046
 CWD="$(realpath $(dirname $0))"
```

Comment:
Agree on this proposal

Reply:
It looks like this is the only remaining requested change, if that's the case can we move forward with this PR and have @knard-intel fix this file when he's working on it.

Reply:
As indicated by @ashleypittman , it will be fixed with the PR https://github.com/daos-stack/daos/pull/13087/files#diff-7d867865cdddd1f9e49de143c074014d98a25c41456c373544d3857493be2a29

Reply:
New version of the CWD fix could be find at https://github.com/daos-stack/daos/pull/13087/files#diff-7d867865cdddd1f9e49de143c074014d98a25c41456c373544d3857493be2a29